cmd/cli/plugin/login/go.mod

```diff
@@ -0,0 +1,132 @@
+module github.com/vmware-tanzu/tanzu-plugins/login
```

Comment:
The different plugin refactoring are using different format for their plugin module name.
Is it ok to be all different for now or should we agree on a format?

For example: 
* #3316 uses `github.com/vmware-tanzu/tanzu-framework/cmd/cli/plugin/package`
* #3284 uses `github.com/vmware-tanzu/tanzu-framework/codegen/plugin`
* #3328 uses `github.com/vwmare-tanzu/tanzu-framework/plugin-admin/test`